build.sh

```diff
@@ -0,0 +1,179 @@
+#! /bin/bash
```

Comment:
I had a look and I think I should be able to make this work for the windows build, using git bash which should have been installed with git. Is that what you had in mind?